.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+
+    container: registry.fedoraproject.org/fedora:latest
+
+    steps:
+      - name: Set up DNF download cache
+        id: dnf-cache
+        uses: actions/cache@v3
+        with:
+          path: /var/cache/dnf
+          key: ${{ runner.os }}-dnfcache
+      - name: Install pre-requisites
+        run: dnf --assumeyes --setopt=install_weak_deps=False install
+              gcc-c++ meson /usr/bin/git /usr/bin/wayland-scanner
+              'pkgconfig(wayland-client)' 'pkgconfig(wayland-protocols)' 'pkgconfig(libpulse-simple)'
+              'pkgconfig(libavutil)' 'pkgconfig(libavcodec)' 'pkgconfig(libavformat)'
+              'pkgconfig(libavdevice)' 'pkgconfig(libavfilter)' 'pkgconfig(libswresample)'
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones speed things up
+      - name: meson configure
+        run: meson ./Build
+      - name: compile with ninja
+        run: ninja -C ./Build -j4
```

Comment:
Does GitHub have more than 1 build job? ninja usually should autodetect the best amount of threads to use.

Reply:
I usually set it to a specific number of cores to make it predictable, but I can drop it to make it dynamic...

Reply:
I think hardcoding 4 threads doesn't make sense for a GitHub Action. AFAIK they are single threaded anyway ..

Reply:
Dropped.